### ISSUE TITLE
add test to check failed output message

### ISSUE THREAD

**Issue by Graur** [objectionary/eo-hamcrest#41]
We need more test to check how failed messages are look like

e.g.

```
[] > throw-test
  assert-that > @
    assert-that 
      2
      $.equal-to 3
    $.equal-to "Expected: 3 equal  to value\n     but: was 2"
```

**Comment by 0pdd** [comment]
@Graur 7 puzzles [#101](https://github.com/objectionary/eo-hamcrest/issues/101), [#102](https://github.com/objectionary/eo-hamcrest/issues/102), [#103](https://github.com/objectionary/eo-hamcrest/issues/103), [#104](https://github.com/objectionary/eo-hamcrest/issues/104), [#105](https://github.com/objectionary/eo-hamcrest/issues/105), [#107](https://github.com/objectionary/eo-hamcrest/issues/107), [#unknown]() are still not solved; solved: [#106](https://github.com/objectionary/eo-hamcrest/issues/106).

**Comment by 0pdd** [comment]
@Graur 6 puzzles [#101](https://github.com/objectionary/eo-hamcrest/issues/101), [#102](https://github.com/objectionary/eo-hamcrest/issues/102), [#103](https://github.com/objectionary/eo-hamcrest/issues/103), [#104](https://github.com/objectionary/eo-hamcrest/issues/104), [#105](https://github.com/objectionary/eo-hamcrest/issues/105), [#unknown]() are still not solved; solved: [#106](https://github.com/objectionary/eo-hamcrest/issues/106), [#107](https://github.com/objectionary/eo-hamcrest/issues/107).

**Comment by 0pdd** [comment]
@Graur 5 puzzles [#101](https://github.com/objectionary/eo-hamcrest/issues/101), [#102](https://github.com/objectionary/eo-hamcrest/issues/102), [#103](https://github.com/objectionary/eo-hamcrest/issues/103), [#104](https://github.com/objectionary/eo-hamcrest/issues/104), [#unknown]() are still not solved; solved: [#105](https://github.com/objectionary/eo-hamcrest/issues/105), [#106](https://github.com/objectionary/eo-hamcrest/issues/106), [#107](https://github.com/objectionary/eo-hamcrest/issues/107).

**Comment by 0pdd** [comment]
@Graur 4 puzzles [#101](https://github.com/objectionary/eo-hamcrest/issues/101), [#102](https://github.com/objectionary/eo-hamcrest/issues/102), [#103](https://github.com/objectionary/eo-hamcrest/issues/103), [#104](https://github.com/objectionary/eo-hamcrest/issues/104) are still not solved; solved: [#105](https://github.com/objectionary/eo-hamcrest/issues/105), [#106](https://github.com/objectionary/eo-hamcrest/issues/106), [#107](https://github.com/objectionary/eo-hamcrest/issues/107), [#unknown]().

**Comment by 0pdd** [comment]
@Graur 3 puzzles [#101](https://github.com/objectionary/eo-hamcrest/issues/101), [#103](https://github.com/objectionary/eo-hamcrest/issues/103), [#104](https://github.com/objectionary/eo-hamcrest/issues/104) are still not solved; solved: [#102](https://github.com/objectionary/eo-hamcrest/issues/102), [#105](https://github.com/objectionary/eo-hamcrest/issues/105), [#106](https://github.com/objectionary/eo-hamcrest/issues/106), [#107](https://github.com/objectionary/eo-hamcrest/issues/107), [#unknown]().

**Comment by 0pdd** [comment]
@Graur 2 puzzles [#101](https://github.com/objectionary/eo-hamcrest/issues/101), [#104](https://github.com/objectionary/eo-hamcrest/issues/104) are still not solved; solved: [#102](https://github.com/objectionary/eo-hamcrest/issues/102), [#103](https://github.com/objectionary/eo-hamcrest/issues/103), [#105](https://github.com/objectionary/eo-hamcrest/issues/105), [#106](https://github.com/objectionary/eo-hamcrest/issues/106), [#107](https://github.com/objectionary/eo-hamcrest/issues/107), [#unknown]().

**Comment by 0pdd** [comment]
@Graur the puzzle [#101](https://github.com/objectionary/eo-hamcrest/issues/101) is still not solved; solved: [#102](https://github.com/objectionary/eo-hamcrest/issues/102), [#103](https://github.com/objectionary/eo-hamcrest/issues/103), [#104](https://github.com/objectionary/eo-hamcrest/issues/104), [#105](https://github.com/objectionary/eo-hamcrest/issues/105), [#106](https://github.com/objectionary/eo-hamcrest/issues/106), [#107](https://github.com/objectionary/eo-hamcrest/issues/107), [#unknown]().

**Comment by 0pdd** [comment]
@Graur all 8 puzzles are solved here: [#101](https://github.com/objectionary/eo-hamcrest/issues/101), [#102](https://github.com/objectionary/eo-hamcrest/issues/102), [#103](https://github.com/objectionary/eo-hamcrest/issues/103), [#104](https://github.com/objectionary/eo-hamcrest/issues/104), [#105](https://github.com/objectionary/eo-hamcrest/issues/105), [#106](https://github.com/objectionary/eo-hamcrest/issues/106), [#107](https://github.com/objectionary/eo-hamcrest/issues/107), [#unknown]().
